### PR TITLE
Standalone/libxml

### PR DESCRIPTION
**Pull request by lmrs2**

### CODE REVIEW

**Comment by inferno-chromium** [comment]
Please rebase on master and fix merge conflicts.

File: fuzzers/afl/fuzzer.py [hunk 1]
```diff
@@ -23,7 +23,8 @@
 
 
 def prepare_build_environment():
-    """Set environment variables used to build AFL-based fuzzers."""
+    """Set environment variables used to build targets for AFL-based
```
Comment (jonathanmetzman) [inline fuzzers/afl/fuzzer.py]: This change shouldn't be here.

File: fuzzers/lafintel/builder.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 Google LLC
```
Comment (jonathanmetzman) [inline fuzzers/lafintel/builder.Dockerfile]: This change shouldn't be in this PR.

File: fuzzers/lafintel/fuzzer.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
+# Copyright 2020 Google LLC
```
Comment (jonathanmetzman) [inline fuzzers/lafintel/fuzzer.py]: This change shouldn't be in this PR.

File: fuzzers/lafintel/runner.Dockerfile [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Google LLC
```
Comment (jonathanmetzman) [inline fuzzers/lafintel/runner.Dockerfile]: This change shouldn't be in this PR.

**Comment by inferno-chromium** [comment]
Triggered CI to run again and verify lafintel stuff fixed.

**Comment by inferno-chromium** [comment]
This breaks laf-intel

```
/afl/afl-clang-fast -pthread -Wl,--no-as-needed -Wl,-ldl -Wl,-lm -Wno-unused-command-line-argument -O3 -fPIC -D__builtin_cpu_supports(x)=0 -D_LARGEFILE64_SOURCE=1 -I. -c -o example.o test/example.c
/afl/afl-clang-fast -pthread -Wl,--no-as-needed -Wl,-ldl -Wl,-lm -Wno-unused-command-line-argument -O3 -fPIC -D__builtin_cpu_supports(x)=0 -D_LARGEFILE64_SOURCE=1  -c -o adler32.o adler32.c
/bin/sh: 1: Syntax error: "(" unexpected
/bin/sh: 1: Syntax error: "(" unexpected
Makefile:160: recipe for target 'adler32.o' failed
make: *** [adler32.o] Error 2
make: *** Waiting for unfinished jobs....
make: *** [example.o] Error 2
Makefile:147: recipe for target 'example.o' failed
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/src/fuzzer.py", line 53, in build
    utils.build_benchmark()
  File "/src/fuzzers/utils.py", line 49, in build_benchmark
    subprocess.check_call(['/bin/bash', '-ex', build_script], env=env)
  File "/usr/lib/python3.5/subprocess.py", line 581, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/bin/bash', '-ex', '/src/build.sh']' returned non-zero exit status 1
```

**Comment by jonathanmetzman** [comment]
@lmrs2 Any plans to land this still?

**Comment by lmrs2** [comment]
Just pushed a fix. I think the error was with zlib, which now compiles.

**Pull request merged**